tests/test_utils.py

```diff
@@ -8,10 +8,10 @@
 class UtilsGetLocalFileContentTestCase(unittest.TestCase):
     def test_unicode_file_content(self):
         content = utils.get_local_file_content("tests/content_unicode.txt")
-        self.assertTrue(isinstance(content, unicode))
+        self.assertTrue(isinstance(content, str))
```

Comment:
Hmm, we may need to do some 2.x/3.x branching, or use something that unites the superset correctly, because `str` is exactly the bug being tested for in 2.x—it needs to parse the file with full unicode representation and store it in a string that has an appropriate encoding.

Looks like we could use [`builtins.str` to get around this](http://python-future.org/compatible_idioms.html#unicode)?

I definitely want to make sure this test is right, though.